### PR TITLE
Update soroban xdr

### PR DESCRIPTION
**Pull request by paulbellamy**
<details>
  <summary>PR Checklist</summary>
  
### PR Structure

* [ ] This PR has reasonably narrow scope (if not, break it down into smaller PRs).
* [ ] This PR avoids mixing refactoring changes with feature changes (split into two PRs
  otherwise).
* [ ] This PR's title starts with name of package that is most changed in the PR, ex.
  `services/friendbot`, or `all` or `doc` if the changes are broad or impact many
  packages.

### Thoroughness

* [ ] This PR adds tests for the most critical parts of the new functionality or fixes.
* [ ] I've updated any docs ([developer docs](https://developers.stellar.org/api/), `.md`
  files, etc... affected by this change). Take a look in the `docs` folder for a given service,
  like [this one](https://github.com/stellar/go/tree/master/services/horizon/internal/docs).

### Release planning

* [ ] I've updated the relevant CHANGELOG ([here](services/horizon/CHANGELOG.md) for Horizon) if
  needed with deprecations, added features, breaking changes, and DB schema changes.
* [ ] I've decided if this PR requires a new major/minor version according to
  [semver](https://semver.org/), or if it's mainly a patch change. The PR is targeted at the next
  release branch if it's not a patch change.
</details>

### What

Updated xdr to commit `026c9cd074bdb28ddde8ee52f2a4502d9e518a09`

Had to change a couple small implementations to adapt.

### Why

OperationEvents change is required for https://github.com/stellar/go/pull/4694

We probably want to update it for the release anyway.

### Todo

- [x] TBD still if we want to include the ScBigInt removal xdr change in this release or delay that until next.
- [x] Handle wasm code/wasm id changes
- [x] Handle InvokeHostFunctionOp.Parameters removal

### CODE REVIEW

**Comment by 2opremio** [comment]
The integration tests should be fixed except for `TestInvokeHostFunctionInvokeStatelessContractFn()` which requires updating https://github.com/stellar/go/blob/soroban-xdr-next/services/horizon/internal/integration/testdata/test_add_u64.wasm  for which I can't find the source code.

I am also going to disable the Soroban RPC tests in this repo, because the fixes/updates required are at the new tools repo ( https://github.com/stellar/soroban-tools/pull/274/ )

File: .github/workflows/horizon.yml [hunk 1]
```diff
@@ -34,8 +34,8 @@ jobs:
     env:
       HORIZON_INTEGRATION_TESTS_ENABLED: true
       HORIZON_INTEGRATION_TESTS_CORE_MAX_SUPPORTED_PROTOCOL: ${{ matrix.protocol-version }}
-      PROTOCOL_20_CORE_DEBIAN_PKG_VERSION: 19.4.1-1097.4e813f20e.focal~soroban
-      PROTOCOL_20_CORE_DOCKER_IMG: sreuland/stellar-core:19.4.1-1097.4e813f20e.focal-soroban
+      PROTOCOL_20_CORE_DEBIAN_PKG_VERSION: 19.5.1-1131.f55f88376.focal~soroban
+      PROTOCOL_20_CORE_DOCKER_IMG: 2opremio/stellar-core:19.5.1-1131.f55f88376.focal-soroban
```
Comment (paulbellamy) [inline .github/workflows/horizon.yml:38]: We should probably have these built somewhere official, not on just our personal dockerhubs... But that's not really an issue with this PR.

**Comment by paulbellamy** [comment]
LGTM, but I can't approve it as I'm an author 😂

File: .github/workflows/horizon.yml (hunk 1 above)
Comment (2opremio) [reply .github/workflows/horizon.yml:38]: Yeah, I didn't find a better way since the docker repo where builds are put is internall.

**Comment by 2opremio** [comment]
It seems like the apt repo is acting up:

> Could not connect to azure.archive.ubuntu.com:80 (52.147.219.192), connection timed out

**Review by tsachiherman — APPROVED** [review]

**Pull request merged**